### PR TITLE
Enable codecov

### PR DESCRIPTION
**Pull request by conbrad**
Enables codecov for our repo: https://app.codecov.io/gh/bcgov/wps/

- Manually ran coverage report for main so branches can be compared right off the bat.
- VSCode auto removes the spaces in `yaml` files please don't make me undo
- We won't get checks until merged into `main` so it has a baseline to compare PR branches to.

### CODE REVIEW

File: codecov.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+comment: false
+threshold: 5%
```
Comment (andrea-williams) [inline codecov.yml:2]: just curious - what are these settings for?

Comment (conbrad) [reply codecov.yml:2]: https://docs.codecov.com/docs/commit-status#branches

**Review by andrea-williams — COMMENTED** [review]
Assuming we're committing to replacing Sonarcloud with CodeCov, we should delete the Sonarcloud badge from our project readme (and replace it with CodeCov if there is a badge?)

Also would be handy to insert links to CodeCov docs in our readme

**Review by andrea-williams — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -82,7 +87,3 @@ Resources:
 - [ZenHub Board](https://app.zenhub.com/workspaces/wildfire-predictive-services-5e321393e038fba5bbe203b8/board?repos=235861506,237125626,237125691)
 - [PEP8](https://github.com/python/peps/blob/master/pep-0008.txt) and [PEP20](https://github.com/python/peps/blob/master/pep-0020.txt) coding conventions, but with 110 character line breaks
 - [Code of Conduct](https://github.com/bcgov/wps/blob/master/CONDUCT.md)
-
```
Comment (Sybrand) [inline README.md:85]: Don't drop this - we'll still keep using sonarcloud, just not for coverage.

File: .github/workflows/integration.yml [hunk 3]
```diff
@@ -197,13 +194,8 @@ jobs:
         working-directory: ./web
         run: yarn run test:ci
 
-      - name: Fix code coverage paths (web)
-        # Due to the way sonarscanner mounts thing when running in docker, we need to change the coverage file.
-        working-directory: ./web/coverage
-        run: sed -i 's/\/home\/runner\/work\/wps\/wps\/web\//\/github\/workspace\//g' lcov.info
-
-      - name: SonarCloud Scan
```
Comment (Sybrand) [inline .github/workflows/integration.yml:205]: We want to keep sonarcloud, just not the code coverage - we can move it to a different job so it happens in parallel though.

File: .github/workflows/integration.yml [hunk 4]
```diff
@@ -6,7 +6,7 @@ on:
       - main
 jobs:
   lint-and-test:
-    name: Lint, Test with Coverage & SonarQube
+    name: Lint, Test with CodeCov
```
Comment (Sybrand) [inline .github/workflows/integration.yml:9]:
```suggestion
    name: Lint, Test, Coverage with CodeCov
```

**Review by Sybrand — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (conbrad) [reply README.md:85]: What do we want to keep sonarcloud for?

Comment (Sybrand) [reply README.md:85]: Does static analysis. Catches security issues, code smells etc. etc.

**Review by Sybrand — CHANGES_REQUESTED** [review]
You have to do the same in static_analysis.yml - that action runs on main after code has been merged.

**Review by Sybrand — APPROVED** [review]

**Pull request merged**